salt/metalk8s/orchestrate/downgrade/init.sls

```diff
@@ -133,6 +133,13 @@ Sync module on salt-master:
     - name: saltutil.sync_all
     - saltenv: metalk8s-{{ dest_version }}
 
+Deploy Kubernetes service config objects:
+  salt.runner:
+  - name: state.orchestrate
+  - mods:
+    - metalk8s.service-configuration.deployed
+  - saltenv: metalk8s-{{ dest_version }}
```

Comment:
You need to add some checks here because you don't want to do this if `dest_version < 2.5.0`

Reply:
Done